crates/wasi-common/wig/src/hostcalls.rs

```diff
@@ -101,33 +99,13 @@ fn generate_wrappers(func: &witx::InterfaceFunc, old: bool) -> TokenStream {
         }
     };
 
-    let c_abi_name = if old {
-        format_ident!("old_wasi_common_{}", name)
-    } else {
-        format_ident!("wasi_common_{}", name)
-    };
-
     quote! {
         pub unsafe fn #name(
             wasi_ctx: &mut super::WasiCtx,
             memory: &mut [u8],
             #(#arg_declarations,)*
-        ) -> #ret {
-            #body
-        }
-
-        #[no_mangle]
-        pub unsafe fn #c_abi_name(
```

Comment:
For referenced, I recommended this get removed since  it needed to get updated anyway and I don't think that anything is using this right now.